### DISCUSSION TITLE
Big new release!

### DISCUSSION THREAD

**Discussion by qwertie** [qwertie/ecsharp#132] [Show and tell]
Hoping someone (@dadhi? @jonathanvdc?) wants to take [v29](https://github.com/qwertie/ecsharp/releases/tag/v2.9.0.2) for a spin - you know, write a macro that does something cool?

**Comment by dadhi** [comment]
@qwertie yesss :)

**Reply by qwertie** [reply]
I'd like to see some small macros that I could put in the documentation as examples. And when I make the online version of LeMP, it's going to need some demos for users to play with...
